docker-compose-dev.yml

```diff
@@ -95,13 +95,24 @@ services:
     depends_on:
       - mysql
     volumes:
+      # shared from wikibase, to run dumpBackup.php and importBackup.php
+      - shared:/var/www/html/:ro
+      - ./mediawiki/LocalSettings.d:/var/www/html/LocalSettings.d
+      - ./mediawiki/extensions/TemplateStyles:/var/www/html/extensions/TemplateStyles
+      - ./mediawiki/extensions/JsonConfig:/var/www/html/extensions/JsonConfig
+      - ./mediawiki/extensions/Math:/var/www/html/extensions/Math
+      - ./mediawiki/extensions/MathSearch:/var/www/html/extensions/MathSearch
+      # dir on host where to store the backups
       - ${BACKUP_DIR:-./backup}:/data
+      # only for running tests
+      - ./test:/test
     environment:
       DB_HOST: mysql.svc # internal docker hostname (alias) of the database service
       DB_NAME: ${DB_NAME}
       DB_USER: ${DB_USER}
       DB_PASS: ${DB_PASS}
       BACKUP_SCHEDULE: ${BACKUP_SCHEDULE:-15 5 * * *}
+      KEEP_DAYS: ${KEEP_DAYS:-100}
```

Comment:
An option to toggle backups on and off would be great, especially if running everything locally.

Reply:
you could set that in a docker-composer.override file